Prüfschritte/de/11.7 Benutzerdefinierte Einstellungen.adoc

```diff
@@ -21,88 +13,68 @@ Im Einzelnen sind dies:
-====
+
+. Einstelungen öffenen (Einstellungne > Bedienungshilfen > Anzeige und Textgröße)
+* "Farben umkehren" aktivieren.
+* App neu starten und prüfen, ob die Einstellungen übernommen werden.
+
+. Einstellungen öffnen (Einstellungen > Anzeige und Hilligeit)
+* Erscheinungsbild "Dunkel" aktivieren.
+* App neu starten und prüfen, ob die Einstellungen übernommen werden.
+
 
 ===== Android
-[%collapsible]
-====
-Folgende Einstellungen unter Einstellungen -> Bedienungshilfen werden durch
+
+. Einstellungen öffnen (Einstellungen > Bedienungshilfen)
 die App unterstützt:
+. Die Einstellungen "Farbumkehr", "Farbkorrektur" und "Dunkles Design" aktivieren.
+. Nach der jeweiligen Einstellung App neu starten und prüfen, ob die Einstellungen übernommen werden.
 
-* Farbumkehr
-* Farbkorrektur
-* Dunkles Design
-====
 
 ==== Schriftart und Schriftgröße
 
-Die Darstellung von Fließtexten richtet sich nach den Einstellungen im System.
-Für die Prüfung werden die systemweiten Einstellungen verändert und die zu
-prüfende App neugestartet.
+Die Darstellung von Fließtexten richtet sich nach den Einstellungen im System. Für die Prüfung werden die systemweiten Einstellungen verändert und die zu prüfende App neugestartet.
 
 ===== iOS / iPadOS
-[%collapsible]
-====
-In iOS / iPadOS müssen folgende Einstellungen unter Einstellungen ->
-Bedienungshilfen -> Anzeige & Textgröße übernommen werden:
-
-* Fetter Text
-* Größerer Text
-====
+. Einstellungen öffnen (Bedienungshilfen > Anzeige & Textgröße)
+. Die Einstellungen "Fetter Text" und "Größerer Text" aktivieren.
+. Nach der jeweiligen Einstellung App neu starten und prüfen, ob die Einstellungen übernommen werden.
 
 ===== Android
-[%collapsible]
-====
-Unter Android müssen folgende Einstellungen in Einstellungen ->
-Bedienungshilfen von der App übernommen werden:
 
-* Schriftgröße
-====
+. Einstellungen öffnen (Einstellungen > Bedienungshilfen)
+. Die Einstellungen "Schriftgröße" und "Fettdruck" aktivieren.
+. App neu starten und prüfen, ob die Einstellungen übernommen werden.
 
 ==== Hinweise
 
-Wenn einer der o. g. Einstellungen in iOS / iPadOS oder Android nicht zu finden
-ist und somit vom Nutzer nicht angepasst werden kann, wird diese Einstellung
-bei der Prüfung nicht berücksichtigt.
+Wenn einer der o. g. Einstellungen in iOS / iPadOS oder Android nicht zu finden ist und somit vom Nutzer nicht angepasst werden kann, wird diese Einstellung bei der Prüfung nicht berücksichtigt.
 
-Die Schriftart und -Größe muss sich nur in Fließtexten an den systemweiten
-Einstellungen richten.
-Es wird nicht verlangt, dass Bedienelemente der App dynamische Veränderungen
-dieser Einstellungen unterstützen, da hier das Layout der App verändert
+Die Schriftart und -Größe muss sich nur in Fließtexten an den systemweiten Einstellungen richten. Es wird nicht verlangt, dass Bedienelemente der App dynamische Veränderungen dieser Einstellungen unterstützen, da hier das Layout der App verändert
```

Comment:
Diese realistische Abgrenzung gibt es normativ ja leider nicht. Das wäre auch ein Thema, was man in BFIT o.ä. diskutieren sollte, wie hoch man hier die Latte wirklich hängen will.